UnitTests/FileDialogTests.cs

```diff
@@ -0,0 +1,319 @@
+using System;
```

Comment:
Put all of this in `./UnitTests/FileServicesTests/`?

Reply:
Done I've put it in `./UnitTests/FileServices` (no Tests suffix) as the other folders don't end with `Tests`.  But I have set the namespace to `Terminal.Gui.FileServicesTests` to be consistent with the rest of the test project:

![image](https://user-images.githubusercontent.com/31306100/230761504-bd7f307b-3a0b-4857-9e81-330028946450.png)